Zend/zend_compile.c

```diff
@@ -6985,7 +6985,7 @@ static zend_type zend_compile_single_typename(zend_ast *ast)
 			uint32_t fetch_type = zend_get_class_fetch_type_ast(ast);
 			if (fetch_type == ZEND_FETCH_CLASS_DEFAULT) {
 				class_name = zend_resolve_class_name_ast(ast);
-				zend_assert_valid_class_name(class_name);
+				zend_assert_valid_class_name(class_name, "a class name");
```

Comment:
Disclaimer: a was unable to figure out how to reach/trigger this code path, and so just always said "class name"

Reply:
Nevermind, found it